electron/execution.ts

```diff
@@ -293,21 +304,19 @@ async function onSetMode(mode) {
   return selector;
 }
 
-async function onLinkExternal(url) {
+async function onLinkExternal(url: string) {
   try {
     await shell.openExternal(url);
   } catch (e) {
     logger.error(e);
   }
 }
 
-function setupListeners() {
-  ipc.answerRenderer('record-journey', recordJourneys);
-  ipc.answerRenderer('run-journey', onTest);
+export default function setupListeners() {
+  ipc.answerRenderer<{ url: string }>('record-journey', recordJourneys);
```

Comment:
```suggestion
  ipc.answerRenderer<RecordJourneyOptions>('record-journey', recordJourneys);
```
Do you think it'd be worth it to make a named type here, and below?